<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>qSlicerWelcomeModuleWidget</class>
  <widget class="qSlicerWidget" name="qSlicerWelcomeModuleWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>577</width>
        <height>910</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Welcome</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
        <widget class="QLabel" name="label">
          <property name="text">
            <string/>
          </property>
          <property name="pixmap">
            <pixmap resource="../qSlicerWelcomeModule.qrc">:/Images/WelcomeLogo.png</pixmap>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QFrame" name="ButtonsFrame">
          <property name="font">
            <font>
              <weight>75</weight>
              <bold>true</bold>
            </font>
          </property>
          <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
            <property name="margin" >
              <number>4</number>
            </property>
            <property name="spacing" >
              <number>4</number>
            </property>
            <item row="0" column="0">
              <widget class="QPushButton" name="LoadDicomDataButton">
                <property name="text">
                  <string>Load DICOM Data</string>
                </property>
                <property name="icon">
                  <iconset resource="../qSlicerWelcomeModule.qrc">
                    <normaloff>:/Images/SlicerLoadDicom-Original.png</normaloff>:/Images/SlicerLoadDicom-Original.png</iconset>
                </property>
                <property name="iconSize">
                  <size>
                    <width>45</width>
                    <height>22</height>
                  </size>
                </property>
              </widget>
            </item>

            <item row="0" column="1">
              <widget class="QPushButton" name="LoadNonDicomDataButton">
                <property name="text">
                  <string>Load Data</string>
                </property>
                <property name="icon">
                  <iconset resource="../qSlicerWelcomeModule.qrc">
                    <normaloff>:/Images/SlicerLoadData-Original.png</normaloff>:/Images/SlicerLoadData-Original.png</iconset>
                </property>
                <property name="iconSize">
                  <size>
                    <width>45</width>
                    <height>22</height>
                  </size>
                </property>
              </widget>
            </item>

            <item row="1" column="1">
              <widget class="QPushButton" name="LoadSampleDataButton">
                <property name="text">
                  <string>Download Sample Data</string>
                </property>
                <property name="icon">
                  <iconset resource="../qSlicerWelcomeModule.qrc">
                    <normaloff>:/Images/SlicerWWW-Original.png</normaloff>:/Images/SlicerWWW-Original.png</iconset>
                </property>
                <property name="iconSize">
                  <size>
                    <width>29</width>
                    <height>22</height>
                  </size>
                </property>
              </widget>
            </item>

            <item row="1" column="0">
              <widget class="QPushButton" name="EditApplicationSettingsButton">
                <property name="text">
                  <string>Customize Slicer</string>
                </property>
                <property name="icon">
                  <iconset resource="../qSlicerWelcomeModule.qrc">
                    <normaloff>:/Images/SlicerAdvancedGear-Original.png</normaloff>:/Images/SlicerAdvancedGear-Original.png</iconset>
                </property>
                <property name="iconSize">
                  <size>
                    <width>22</width>
                    <height>22</height>
                  </size>
                </property>
              </widget>
            </item>

          </layout>
        </widget>
      </item>

      <item>
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="FeedbackCollapsibleWidget">
          <property name="text">
            <string>Feedback</string>
          </property>
          <property name="collapsed">
            <bool>false</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowser_13">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/Feedback.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="WelcomeAndAboutCollapsibleWidget">
          <property name="text">
            <string>About</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowse">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/About.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="OverviewCollapsibleWidget">
          <property name="text">
            <string>The Main Window</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowser_2">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/Overview.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="LoadingScenesAndDataCollapsibleWidget">
          <property name="text">
            <string>Loading and Saving</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowser_4">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/LoadingScenesAndData.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="AdjustingDataDisplayCollapsibleWidget">
          <property name="text">
            <string>Display</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowser_5">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/AdjustingDataDisplay.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="MouseModesCollapsibleWidget">
          <property name="text">
            <string>Mouse &amp;&amp; Keyboard</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowser_7">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/MouseModes.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="OtherUsefulHintsCollapsibleWidget">
          <property name="text">
            <string>Documentation &amp;&amp; Tutorials</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowser_9">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/OtherUsefulHints.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton" name="AcknowledgmentCollapsibleWidget">
          <property name="text">
            <string>Acknowledgment</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="contentsFrameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="margin">
              <number>0</number>
            </property>
            <item>
              <widget class="ctkFittedTextBrowser" name="textBrowser_11">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="source">
                  <url>
                    <string>qrc:HTML/Acknowledgment.html</string>
                  </url>
                </property>
                <property name="openExternalLinks">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="VerticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>0</width>
              <height>0</height>
            </size>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="0" margin="0"/>
  <customwidgets>
    <customwidget>
      <class>ctkCollapsibleButton</class>
      <extends>QWidget</extends>
      <header>ctkCollapsibleButton.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkFittedTextBrowser</class>
      <extends>QTextBrowser</extends>
      <header>ctkFittedTextBrowser.h</header>
    </customwidget>
    <customwidget>
      <class>qSlicerWidget</class>
      <extends>QWidget</extends>
      <header>qSlicerWidget.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources>
    <include location="../qSlicerWelcomeModule.qrc"/>
    <include location="../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc"/>
    <include location="../../../../../Resources/qSlicer.qrc"/>
  </resources>
</ui>
